compiler/plugins/target/ROCM/ROCMTarget.cpp

```diff
@@ -702,8 +702,8 @@ struct ROCMSession final
     : PluginSession<ROCMSession, ROCmOptions,
                     PluginActivationPolicy::DefaultActivated> {
   void populateHALTargetDevices(IREE::HAL::TargetDeviceList &targets) {
-    // #hal.device.target<"rocm", ...
-    targets.add("rocm",
+    // #hal.device.target<"hip", ...
+    targets.add("hip",
                 [&]() { return std::make_shared<ROCMTargetDevice>(options); });
```

Comment:
Is this going to break anyone using `--iree-hal-target-backends=rocm`, or is that lines 710-711 in this file?
```c++
    // #hal.executable.target<"rocm", ...
    targets.add("rocm", [&]() {
```

Reply:
correct this is just for the devices flag - backends will still be rocm